packages/mock/src/MockStore.ts

```diff
@@ -301,6 +301,7 @@ export class MockStore implements IMockStore {
     }
 
     const nullableFieldType = getNullableType(fieldType);
+    if (value === null) return null;
```

Comment:
Can this return be added to line 300, where there is already check for `value === null`?

```ts
   if (value === null) {
      if (!isNullableType(fieldType)) {
        throw new Error(`should not be null because ${fieldTypeName} is not nullable. Received null.`);
      }
      return null;
    }
```

Reply:
Makes sense!